docker-compose.yml

```diff
@@ -51,6 +51,13 @@ services:
       - SUPPORT_ACTIVITY_LOG=${SUPPORT_ACTIVITY_LOG}
       - DEBUG=express-session
       - LOCALSTACK_HOSTNAME=host.docker.internal
+      - SUPPORT_TRIAGE_PAGE=${SUPPORT_TRIAGE_PAGE}
```

Comment:
I don't think these need to be in the docker compose if they are in your local .env file

Reply:
You're right yes. Changes reverted.